server/Cargo.toml

```diff
@@ -19,11 +19,12 @@ chrono = { version = "0.4.7", features = ["serde"] }
 serde_json = { version = "1.0.52", features = ["preserve_order"]}
 failure = "0.1.8"
 serde = { version = "1.0.105", features = ["derive"] }
-actix = "0.9.0"
-actix-web = "2.0.0"
-actix-files = "0.2.1"
-actix-web-actors = "2.0.0"
+actix = "0.10.0-alpha.2"
+actix-web = { version = "3.0.0-alpha.3", features = ["rustls"] }
```

Comment:
Is rustls for the http client? Cause otherwise we dont need it.

Reply:
yeah, either the "rustls" feature or the "openssl" feature needs to be enabled to allow the client to hit HTTPS endpoints